extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -226,20 +229,32 @@ void ConsumeWindowsEventLog::onSchedule(const std::shared_ptr<core::ProcessConte
     }
   }
 
+  std::string mode;
+  context->getProperty(OutputFormat.getName(), mode);
+
+  writeXML_ = (mode == Both || mode == XML);
+
+  writePlainText_ = (mode == Both || mode == Plaintext);
+
+  if (writeXML_) {
+    char systemDir[MAX_PATH];
+    if (GetSystemDirectory(systemDir, sizeof(systemDir))) {
+      hMsobjsDll_ = LoadLibrary((systemDir + std::string("\\msobjs.dll")).c_str());
```

Comment:
Having this in onSchedule and `FreeLibrary` in the destructor means that if onSchedule is called multiple times (configuration reload, for example), the reference counter for this DLL will be incremented multiple times and it won't be released when the object is finally destructed.

Reply:
Fixed.